cmd/devp2p/README.md

```diff
@@ -96,7 +96,7 @@ geth --datadir <datadir> --nodiscover --nat=none --networkid 19763 --verbosity 5
 
 Then, run the following command, replacing `<enode ID>` with the enode of the geth node: 
  ```
- devp2p rlpx eth-test <enode ID> cmd/devp2p/internal/ethtest/testdata/fullchain.rlp cmd/devp2p/internal/ethtest/testdata/genesis.json
+ devp2p rlpx eth-test <enode ID> cmd/devp2p/internal/ethtest/testdata/chain.rlp cmd/devp2p/internal/ethtest/testdata/genesis.json
 ```
```

Comment:
Would be nice to have a sentence here that says the node needs to be reset to the initial state before running the test again.